### DISCUSSION TITLE
Subclustering: is it necessary to create a new Arrow file?

### DISCUSSION THREAD

**Discussion by macpheke** [GreenleafLab/ArchR#685] [Questions / Documentation]
Is there anything wrong with having multiple ArchRProject objects from the same ArrowFile?

Like if I have an ArchRProject object `proj` with 10,000 cells, then I do:
`proj_1 <- subsetCells(ArchRProject = proj, cellNames = group_A)`
`proj_2 <- subsetCells(ArchRProject = proj, cellNames = group_B)`

Will it create problems for my Arrow file down the line if I then run `addIterativeLSI()`, `addUMAP()`, and `addImputeWeights()` on proj, proj_1, and proj_2?

Or is it truly necessary to keep independent project folders with independent arrow files for each?

**Comment by rcorces** [comment]
This is not a supported workflow. Imagine you use `proj_1` to perform an action that modifies the arrow files. `proj_2` will not expect that modification and things will break.
